### PR TITLE
Adding two single/simple/demo node deployments for persistent/ephemeral

### PR DESCRIPTION
**Pull request by matzew**
### Type of change

- Enhancement / new feature
- Documentation

### Description

It's adding simple installs for both storage types, but just using one node for each, Apache Zookeeper and Apache Kafka.

Especially with demos this is handy

### CODE REVIEW

**Comment by matzew** [comment]
@scholzj @ppatierno feel free to ignore

File: examples/kafka/kafka-ephemeral-single.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    config:
+      offsets.topic.replication.factor: 3
+      transaction.state.log.replication.factor: 3
+      transaction.state.log.min.isr: 2
+    storage:
+      type: ephemeral
+  zookeeper:
+    replicas: 1
+    storage:
+      type: ephemeral
```
Comment (scholzj) [inline examples/kafka/kafka-ephemeral-single.yaml]: This really doesn't work - you cannot have ephemeral Zoo or Kafka with single node. They will loose all data with every restarts / rolling update. My suggestion is to remove the ephemeral version completely.

File: examples/kafka/kafka-persistent-single.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 1
+    listeners:
+      plain: {}
+      tls: {}
+    config:
+      offsets.topic.replication.factor: 3
```
Comment (scholzj) [inline examples/kafka/kafka-persistent-single.yaml]: All these options should be set to 1. With single Kafka broker, you cannot have anything with replication factor and ISR bigger then 1.

**Comment by matzew** [comment]
@scholzj @ppatierno  done

**Pull request merged**